changelog/new_add_new_rspecfilefixture_cop.md

```diff
@@ -0,0 +1 @@
+* [#1214](https://github.com/rubocop/rubocop-rails/pull/1214): Add new `RSpec/FileFixture` cop. ([@leoarnold][])
```

Comment:
We're still using the old approach with just one CHANGELOG.md in this repo - it's not a major pain to merge it.